Makefile.org

```diff
@@ -418,12 +418,22 @@ clean:	libclean
 	rm -f $(LIBS)
 	rm -f openssl.pc libssl.pc libcrypto.pc
 	rm -f speed.* .pure
+	$(RM) test/newkey.pem test/testkey.pem test/testreq.pem
+	-$(RM) `find . -name '*.o' -a \! -path "./.git/*"`
```

Comment:
@dot-asm has a point, though.  The cleanup of `test/{whatever}` should happen in `test/Makefile` rather than the top `Makefile`, in 1.0.2